### PR TITLE
change default dtype to int

### PR DESCRIPTION
**Pull request by luiztauffer**
float values can be obtained with gain

### CODE REVIEW

**Comment by luiztauffer** [comment]
@alejoe91 I changed the default dtype to int16, we can use the gain value to get traces in float (as it was before). The RecordingExtractor does that already, I'll check if that needs any update.
However, I think there's something wrong with the values in Volts obtained with int16 * 0.195 (the fixed value for gain in OpenEphys), since the numerical values are fluctuating between +/- 300

If we scale it dividing by 32768 (2**16 / 2), the values seem to return to typical ecephys voltage ranges

**Comment by alejoe91** [comment]
Hi Luiz,

I double checked and multiplying the ADC value by the bit_volts value returns the uV value (https://groups.google.com/g/open-ephys/c/9CKgVPoEF7M?pli=1). 

Note that +/- 300 uV is perfectlu fine if the data is raw (contains LFP). These are typical ecephys voltage ranges.

I agree we could use `int16` by default for the Continuous format

File: pyopenephys/core.py [hunk 1]
```diff
@@ -649,7 +649,8 @@ def _read_tracking(self):
 
     def _read_analog_signals(self):
         if self.experiment.acquisition_system is not None:
-            gain = 0.195 # this is fixed in the open ephys system
+            # fixed value in the open ephys system / bits-to-Volts A/D scale
+            gain = 0.195 / 2**15
```
Comment (alejoe91) [inline pyopenephys/core.py]:
```suggestion
            gain = 0.195
```

File: pyopenephys/openephys_tools.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 MAX_NUMBER_OF_RECORDS = int(1e6)
 MAX_NUMBER_OF_EVENTS = int(1e6)
 
-def loadContinuous(filepath, dtype=float):
+def loadContinuous(filepath, dtype=np.int16):
```
Comment (alejoe91) [inline pyopenephys/openephys_tools.py]: I think it's safer to have these as strings
```suggestion
def loadContinuous(filepath, dtype='int16'):  
```

File: pyopenephys/openephys_tools.py [hunk 3]
```diff
@@ -17,9 +17,9 @@
 MAX_NUMBER_OF_RECORDS = int(1e6)
 MAX_NUMBER_OF_EVENTS = int(1e6)
 
-def loadContinuous(filepath, dtype=float):
-    assert dtype in (float, np.int16), \
-        'Invalid data type specified for loadContinous, valid types are float and np.int16'
+def loadContinuous(filepath, dtype='int16'):  
+    assert dtype in (float, 'int16'), \
```
Comment (alejoe91) [inline pyopenephys/openephys_tools.py]: Can you make float also a str?

**Review by alejoe91 — CHANGES_REQUESTED** [review]

File: pyopenephys/openephys_tools.py (hunk 3 above)
Comment (luiztauffer) [reply pyopenephys/openephys_tools.py]: sure

Comment (luiztauffer) [reply pyopenephys/openephys_tools.py]: done

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**